SocialNetworkApi/DataAccess/Entities/Comment.cs

```diff
@@ -1,17 +1,17 @@
-namespace SocialNetwork.Entities
+namespace SocialNetworkApi.DataAccess.Entities;
+
+public class Comment
 {
-    public class Comment
-    {
-        public Guid Id { get; set; }
-        public Guid PostId { get; set; }
-        public Guid UserId { get; set; }
-        public string Content { get; set; }
-        public DateTime CreatedAt { get; set; }
+    public Guid Id { get; set; }
+    public Guid PostId { get; set; }
+    public Guid UserId { get; set; }
+    public string Content { get; set; }
+    public DateTime CreatedAt { get; set; }
 
-        public Comment()
-        {
-            Id = Guid.NewGuid();
-            CreatedAt = DateTime.UtcNow;
-        }
+    public Comment()
```

Comment:
Non-nullable property 'Content' must contain a non-null value when exiting constructor. Consider adding the 'required' modifier or declaring the property as nullable.